cart/cart.css

```diff
@@ -0,0 +1,156 @@
+@import url("../css/font.css");
+@import url("../css/nav.css");
+@import url("../css/card.css");
+/*Elements*/
+html {
+  width: 100%;
+  font-size: 8px;
+  font-family: Philosophy, "Noto Sans";
+}
+body {
+  height: 100vh;
+  border: 0;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+}
+nav {
+  display: flex;
+  flex-direction: row;
+  min-height: 10vh;
+  width: 100%;
+  align-items: center;
+  backdrop-filter: blur(10px);
+  box-shadow: -0px 0px 20px 6px rgba(0, 0, 0, 0.4);
```

Comment:
If you're using the box-shadow throughout your app then you can use CSS variables here and put this in the main.css file